3-box-model/styles/styles.css

```
@@ -0,0 +1,31 @@
+header {
+    color: white;
+    background-color: black;
+    border: 3px solid green;
+    position: sticky;
+}
+
+#menu {
```

avoid the id selector, use classnames so they can be reused.

```suggestion
.menu {
```